src/plugins/interactive_setup/public/app.tsx

```diff
@@ -6,22 +6,59 @@
  * Side Public License, v 1.
  */
 
-import { EuiPageTemplate, EuiPanel, EuiText } from '@elastic/eui';
-import React from 'react';
+import './app.scss';
+
+import { EuiIcon, EuiPanel, EuiSpacer, EuiTitle } from '@elastic/eui';
+import type { FunctionComponent } from 'react';
+import React, { useState } from 'react';
+
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { EnrollmentTokenForm } from './enrollment_token_form';
+import { ManualConfigurationForm } from './manual_configuration_form';
+import { ProgressIndicator } from './progress_indicator';
+
+export const App: FunctionComponent = () => {
+  const [page, setPage] = useState<'token' | 'manual' | 'success'>('token');
 
-export const App = () => {
   return (
-    <EuiPageTemplate
-      restrictWidth={false}
-      template="empty"
-      pageHeader={{
-        iconType: 'logoElastic',
-        pageTitle: 'Welcome to Elastic',
-      }}
-    >
-      <EuiPanel>
-        <EuiText>Kibana server is not ready yet.</EuiText>
-      </EuiPanel>
-    </EuiPageTemplate>
+    <div className="interactiveSetup login-form">
+      <header className="interactiveSetup__header">
+        <div className="interactiveSetup__content eui-textCenter">
+          <EuiSpacer size="xxl" />
+          <span className="interactiveSetup__logo">
+            <EuiIcon type="logoElastic" size="xxl" />
+          </span>
+          <EuiTitle size="m" className="interactiveSetup__title">
+            <h1>
+              <FormattedMessage
+                id="interactiveSetup.app.pageTitle"
+                defaultMessage="Configure Elastic to get started"
+              />
+            </h1>
+          </EuiTitle>
+          <EuiSpacer size="xl" />
+        </div>
+      </header>
+      <div className="interactiveSetup__content interactiveSetup-body">
+        <EuiPanel paddingSize="l">
+          <div hidden={page !== 'token'}>
```

Comment:
Yeh, I'm using hidden on purpose here to retain the form state when toggling back and forth between enrollment token and manual configuration forms. It would be weird UX to lose what you typed.

Reply:
Ah, makes sense, thanks for clarifying!